src/services/schedule.ts

```diff
@@ -63,6 +63,10 @@ function getMeetingDays(course: CourseSchedule): string[] {
   return dayArray;
 }
 
+const asdf = moment('2022-06-10T00:00:00');
+const month = asdf.month();
+console.log(month);
```

Comment:
```suggestion
const asdf = moment('2022-06-10T00:00:00');
const month = asdf.month();
console.log(month);
```
im assuming this is testing. remember to remove it